Examples/Examples/FormViewExampleView.swift

```diff
@@ -67,6 +74,15 @@ struct FormViewExampleView: View {
                         .padding([.horizontal])
                 }
             
+                .alert("Cancel editing?", isPresented: $cancelConfirmationIsShowing) {
+                    Button("Cancel editing", role: .destructive) {
+                        formViewModel.undoEdits()
+                        featureForm = nil
+                        isPresented = false
```

Comment:
Could we have a `didSet` block on the `featureForm` property which set `isPresented`? That might be more clear. In that case I would also change the `Cancel` button block, [L96](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/2fcb74eaa53f9ea6982c21a6da228da82ac15c4e/Examples/Examples/FormViewExampleView.swift#L96) to just set `featureForm = nil`.

And the same for [L106](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/2fcb74eaa53f9ea6982c21a6da228da82ac15c4e/Examples/Examples/FormViewExampleView.swift#L106)